From 7bb958f8db1af43c97cc3345aada5d6f3231c2a4 Mon Sep 17 00:00:00 2001
From: Tony Mak <tonymak@google.com>
Date: Wed, 24 Jan 2018 15:11:06 +0000
Subject: [PATCH 071/306] Swap the order of realCallingUid and realCallingPid
 in startActivityInPackage

Checked ag/3483468, we are always passing pid first, let's swap the
order then.

Test: Setup work profile -> add work account

Fix: 72429434

Change-Id: I8fd05b42fa22a3e46a87b3f6a0902e0e5a797dc9
(cherry picked from commit 793e14d094c1534c8aa78741734f8b274ca2f1cf)
---
 .../java/com/android/server/am/ActivityStartController.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/am/ActivityStartController.java b/services/core/java/com/android/server/am/ActivityStartController.java
index f9932b20fb5..5551914f706 100644
--- a/services/core/java/com/android/server/am/ActivityStartController.java
+++ b/services/core/java/com/android/server/am/ActivityStartController.java
@@ -220,7 +220,7 @@ public class ActivityStartController {
         }
     }
 
-    final int startActivityInPackage(int uid, int realCallingUid, int realCallingPid,
+    final int startActivityInPackage(int uid, int realCallingPid, int realCallingUid,
             String callingPackage, Intent intent, String resolvedType, IBinder resultTo,
             String resultWho, int requestCode, int startFlags, SafeActivityOptions options,
             int userId, TaskRecord inTask, String reason) {
-- 
2.17.1

